doc/classes/OS.xml

```diff
@@ -653,7 +653,7 @@
 			<param index="0" name="name" type="String" />
 			<description>
 				Requests permission from the OS for the given [param name]. Returns [code]true[/code] if the permission has been successfully granted.
-				[b]Note:[/b] This method is currently only implemented on Android, to specifically request permission for [code]"RECORD_AUDIO"[/code] by [code]AudioDriverOpenSL[/code].
+				[b]Note:[/b] This method is currently only implemented on Android and is used by [code]AudioDriverOboe[/code].
```

Comment:
I think that the specific permissions and what they are for should be put somewhere else? It would feel better if they were in AudioServer docs somewhere, where the user would be expected to look first for any audio specific questions and issues. This PR also adds one more permission "MODIFY_AUDIO_SETTINGS" for SCO support on older devices, but I'm out of ideas where to put it.